libs/agno/agno/tools/smallest_ai.py

```diff
@@ -0,0 +1,307 @@
+from base64 import b64encode
+from io import BytesIO
+from os import getenv, path
+from pathlib import Path
+from typing import Literal, Optional
+from uuid import uuid4
+
+from agno.agent import Agent
+from agno.media import AudioArtifact
+from agno.tools import Toolkit
+from agno.utils.log import logger
+
+try:
+    from smallest import Smallest  # type: ignore
```

Comment:
@Itz-Antaripa even after installing `smallestai` the import doesn't seem to work-

After running cookbook, says this-
![image](https://github.com/user-attachments/assets/e3255455-4ca0-42be-8794-3d2e51eac006)

Can you confirm if their api has changed or something like that?

Maybe its the below one, i'm not sure-
![image](https://github.com/user-attachments/assets/4f6df2af-b817-4b17-8088-e8093b39c9e2)